### PR TITLE
Replacing deprecated components

### PR DESCRIPTION
**Pull request by siggakogk**
# Deprecated cleanup

Replacing deprecated components in the web project for consistency
Fixing process entry bug

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/web/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -119,35 +120,51 @@ export const Modal: FC<ModalProps> = ({
                 borderRadius="large"
                 className={styles.content}
               >
-                <button onClick={onCloseEvent} className={styles.close}>
-                  <Icon type="close" color="blue400" width="18" height="18" />
-                </button>
+                <Box className={styles.close}>
```
Comment (jeremybarbet) [inline apps/web/components/Modal/Modal.tsx:123]: Not sure if the modal/dialog has the same design, but we now have one component for that in the UI https://ui.devland.is/?path=/docs/components-dialogprompt--default

**Review by jeremybarbet — APPROVED** [review]
LGTM, just one thing about the modal component, but not sure if it's the same and you want to do it in this PR

File: apps/web/components/Modal/Modal.tsx (hunk 1 above)
Comment (siggakogk) [reply apps/web/components/Modal/Modal.tsx:123]: It's absolutely the same design! Had no idea :D I'm going to switch this modal out

**Review by DavidBachmann — APPROVED** [review]
👍

**Pull request merged**